src/rgb/standards.rs

```diff
@@ -0,0 +1,53 @@
+//!Various RGB standards.
+
+use num::Float;
+use Yxy;
+use flt;
+use white_point::{WhitePoint, D65};
+use rgb::{Primaries, RgbSpace, RgbStandard};
+use pixel::TransferFn;
+
+///The sRGB color space.
+#[derive(Copy, Clone, Debug, PartialEq, Eq)]
+pub struct Srgb;
+
+impl Primaries for Srgb {
```

Comment:
Ok, I have read some of it now, but I think I get the point. It describes how to adapt the sRGB primaries to D50, to prepare them for being included in an ICC profile. The point of that is, as far as I know, to preserve their perceived colors in relation to what's white. That doesn't _necessarily_ mean that we have to do the same with `Primaries`, unless we want them to be perceptively connected to some white point. We may also choose to not do that, and only care about their numerical values (preserving xyY, but losing the perceptive values). The question is what's best when representing the primaries as a type, like this. Chromatic adaption is a bit runtime expensive, but what's the alternative if you want an ICC adapted space?